config.js

```diff
@@ -18,6 +18,7 @@
     "featureManagerIsSecurityLevelEnabled": true,
     "featureManagerIsSSLEnabled": false,
     "featureManagerIsWAFEnabled": true,
+    "isSubdomainCheckEnabled": true,
```

Comment:
Don't worry about changing it now but in the future lets default features that won't be used by all plugins to `false`.